### PR TITLE
SpeedyNodes

### PR DESCRIPTION
**Pull request by pikansj0s**
Updated list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,6 +107,16 @@ The nodes are deployed in multiple regions around the globe using multi-cloud, h
 
 Checkout: [chainstack.com](https://chainstack.com/)
 
+## SpeedyNodes
```
Comment (arddluma) [inline README.md:110]: Please remove this

File: README.md [hunk 2]
```diff
@@ -107,6 +107,7 @@ The nodes are deployed in multiple regions around the globe using multi-cloud, h
 
 Checkout: [chainstack.com](https://chainstack.com/)
 
+
```
Comment (arddluma) [inline README.md:110]: Could you pls remove this line?

Comment (pikansj0s) [reply README.md]: Done

**Comment by arddluma** [comment]
LGTM, thanks for your contribution!

**Pull request merged**